source/typescript.txt

```diff
@@ -0,0 +1,35 @@
+.. _typescript-center:
+
+==========
+TypeScript
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+Introduction
+------------
+
+Welcome to the documentation site for official MongoDB TypeScript support.
+Since Typescript builds on JavaScript, you can use the
+`Node.js Driver <https://www.mongodb.com/docs/drivers/node/current/>`__
```

Comment:
Q: On a similar note, is there a source constant for "Node.js Driver"? We use those quite frequently in other repos and might be worth doing the same here.